components/application-connectivity-validator/cmd/applicationconnectivityvalidator/applicationconnectivityvalidator.go

```diff
@@ -7,13 +7,18 @@ import (
 	"sync"
 	"time"
 
+	"github.com/patrickmn/go-cache"
+	log "github.com/sirupsen/logrus"
+
+	// allow client authentication against GKE clusters
+	//_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
dead imports?

Reply:
I see that import often commented in k8s related projects because it's easy to uncomment and test directly against GCE. I'm neutral about it and can as well remove it.

Reply:
👍 As long as it's consistent